src/Deprecated12/ClassOrganization.extension.st

```diff
@@ -105,7 +105,7 @@ ClassOrganization >> renameCategory: oldName toBe: newName [
 ClassOrganization >> silentlyRenameCategory: oldName toBe: newName [
 
 	self
-		deprecated: 'Use #silentlyRenameProtocolNamed:toBe: instead.'
-		transformWith: '`@rcv silentlyRenameCategory: `@arg1 toBe: `@arg2' -> '`@rcv silentlyRenameProtocolNamed: `@arg1 toBe: `@arg2'.
-	self silentlyRenameProtocolNamed: oldName toBe: newName
+		deprecated: 'Use #renameProtocolNamed:toBe: instead.'
```

Comment:
Maybe a little comment to this method to explain why a user should not use `silentlyRenameProtocolNamed:`?

Reply:
Done!